lansa-vmss-windows-autoscale-sql-database/azuredeploy.json

```diff
@@ -0,0 +1,829 @@
+                        }
+                     }
+                  ]
+               },
+               "extensionProfile": {
+                  "extensions": [
+                     {
+                        "name": "FirstInstall",
+                        "properties": {
+                           "publisher": "Microsoft.Compute",
+                           "type": "CustomScriptExtension",
+                           "typeHandlerVersion": "1.8",
+                           "autoUpgradeMinorVersion": "true",
+                           "forceUpdateTag": "[concat( parameters('installMSI'),parameters('updateMSI'),parameters('triggerWebConfig'),parameters('UninstallMSI'),parameters('fixLicense'))]",
+                           "settings": {
+                              "fileUris": [ "[variables('gitRefreshUri')]" ],
+                              "commandToExecute": "[concat('powershell -NoProfile -ExecutionPolicy unrestricted -command \"& {pushd;./',variables('gitRefreshName'), ' ', parameters('gitBranch'), ';popd;', 'pushd;c:\\lansa\\scripts\\azure-custom-script.ps1 ', '-server_name \"', reference(variables('sqlserverName')).fullyQualifiedDomainName, '\"', ' -DBUT \"SQLAZURE\"',  ' -dbname \"', parameters('databaseName'), '\"', ' -dbuser \"', parameters('databaseLogin'), '\"', ' -dbpassword \"', parameters('databaseLoginPassword'), '\"', ' -webuser \"', parameters('webUsername'), '\"', ' -webpassword \"', parameters('webPassword'), '\"', ' -MSIuri \"', parameters('msiUri'), '\"',  ' -maxconnections \"', parameters('webServerMaxConnect'), '\"', ' -trace \"', parameters('trace'), '\"', ' -installMSI \"', parameters('installMSI'), '\"', ' -updateMSI \"', parameters('updateMSI'), '\"',' -triggerWebConfig \"', parameters('triggerWebConfig'), '\"',' -UninstallMSI \"', parameters('UninstallMSI'), '\"', ' -fixLicense \"', parameters('fixLicense'), '\"', ';if ($LASTEXITCODE -ne 0) {Write-Error (\"MSI Install failed\");exit $LASTEXITCODE}; exit 0;popd;}\"')]"
+                           }
+                        }
+                     }
+                  ]
+               }
+            }
+         }
+      },
+      {
+         "type": "Microsoft.Insights/autoscaleSettings",
+         "apiVersion": "2015-04-01",
+         "name": "autoscalehostdb",
+         "location": "[resourceGroup().location]",
+         "dependsOn": [
+            "[concat('Microsoft.Compute/virtualMachineScaleSets/', variables('dbvmssName'))]"
+         ],
+         "properties": {
+            "name": "autoscalehostdb",
+            "targetResourceUri": "[concat('/subscriptions/',subscription().subscriptionId, '/resourceGroups/',  resourceGroup().name, '/providers/Microsoft.Compute/virtualMachineScaleSets/', variables('dbvmssName'))]",
+            "enabled": true,
+            "profiles": [
+               {
+                  "name": "Profile1",
+                  "capacity": {
+                     "minimum": "1",
+                     "maximum": "1",
```

Comment:
So, you can have a scale set without autoscale. Adding autoscale with min=max does nothing except make the template more confusing. If the VM host dies, the scale set will automatically move the VM to a new host without autoscale.